CHANGES.md

```diff
@@ -0,0 +1,22 @@
+## Release History
+
+### Current Release
+
+#### Update Note for 0.3.0
+ - XML format has been changed
+   - New container element "abandon:journal"
+   - Renamed elemet "txn" to "posting"
```

Comment:
We are using `post` everywhere in abandon. `posting` is what ledger seems to use, but I find it too long and odd. Also, "posting" in accounting seems to have [its own meanings](http://www.accountingtools.com/questions-and-answers/what-is-posting-in-accounting.html). To make matters worse, in English, both "post" and "posting" can be a noun or a verb.

I think using `post` will be better in this PR, to be consistent with rest of abandon. If we strongly feel about changing to `posting`, let's do it separately.